cothority.go

```diff
@@ -80,15 +80,20 @@ func main() {
 			Flags: serverFlags,
 		},
 		{
-			Name:    "check",
-			Aliases: []string{"c"},
-			Usage:   "Check if the servers in the group definition are up and running",
-			Action:  checkConfig,
+			Name:      "check",
+			Aliases:   []string{"c"},
+			Usage:     "Check if the servers in the group definition are up and running",
+			ArgsUsage: "Cothority group definition file",
+			Action:    checkConfig,
 			Flags: []cli.Flag{
 				cli.StringFlag{
 					Name:  "g",
 					Usage: "Cothority group definition file",
 				},
+				cli.BoolFlag{
+					Name:  "detail,l",
+					Usage: "Show detail in signing",
```

Comment:
More detail for detail ^^  Namely, it will try pair wise servers together or stg like that.